### ISSUE TITLE
Home button

### ISSUE THREAD

**Issue by karanharshey** [Abhay-hack/Lumina#6]
Add a home button which will take you to the main screen. It will be convenient when one wishes to jump to the home screen after visiting multiple sections such as job, internship, etc.

**Comment by Abhay-hack** [comment]
Yeah its a good. I'm assigning it to you. But i can assign labels after 1st of Jan. Feel free to ask further questions.

**Comment by prakhar123561** [comment]
Could you assign this to me ? I can also contribute it whichever you like you can merge pull request?
